README.md

```diff
@@ -131,6 +131,27 @@ argument to see all DeepState options.
 
 If you want to use DeepState in C/C++ code, you will likely want to run `sudo make install` from the `$DEEPSTATE/build` directory as well.  The examples mentioned below (file system, databases) assume this has already been done.
 
+### Docker
+
+You can also try out Deepstate with Docker.
+
+```bash
+# Run container with a shared examples/ directory
+# Note that `--rm` will make the container be deleted if you exit it
+# (if you want to persist data from the container, use docker volumes)
+# (we need to increase maximum stack size, so we use ulimit for that)
+$ docker run --rm -it --ulimit stack=100000000:100000000 trailofbits/deepstate bash
```

Comment:
The `ulimit` specification may be unnecessary but I figured it would be a best practice based on the suggestion for the Manticore docker image to specify this ulimit

Reply:
Yes, this is a good idea to have.